### PR TITLE
"NoPrintConversion" functional option

### PR DESCRIPTION
**Pull request by Kjeldgaard**
New function which adds initial arguments to be parsed to exiftool.

### CODE REVIEW

**Comment by barasher** [comment]
Hi @Kjeldgaard, thanks for the PR !

This feature is bothering me:
- :+1:  on one side, it enabled exiftool tuning to fit the user's need
- :-1: on the other side, it can completely break `go-exiftool` if `exiftool` behaviour changes too much
For instance, if the provided parameters change the extracted metadata serialization in a way that `go-exiftool` does not support, the library will behave incorrectly

`Exiftool` supports so many parameters that it will be really hard to test and filter.

But since I consider that this feature might interest some users, let's do this this way:
- add some unit tests to your code
- tell clearly in the godoc comments that this feature is "dangerous" (unsafe) if not well mastered, that extra parameter might break the library and that most of the users should not use this feature
- add a paragraph `Advanced usage` in the README.MD and give a real example for this functional option

**Comment by Kjeldgaard** [comment]
Hi barasher,

Thanks for your comment. I totalky understand your concerns about this change, as it could break go-exiftool functionality. Since it is actually only the "-n" argument I need, I have come to the conclusion that it would probably be better to achieve this similarly to the charset implementation. What do you think?

**Comment by barasher** [comment]
I've taken a look at exiftool's `-n` parameter : I agree with you, a dedicated "functional option" seems to be the best option.
Deal ! Can you handle it ?

**Comment by Kjeldgaard** [comment]
I've made a new dedicated function for 'No Print Conversion'. What do you think?

File: exiftool.go [hunk 1]
```diff
@@ -184,4 +184,14 @@ func Charset(charset string) func(*Exiftool) error {
 		e.extraInitArgs = append(e.extraInitArgs, "-charset", charset)
 		return nil
 	}
-}
\ No newline at end of file
+}
+
+// Function to use 'No print conversion' mode, see https://exiftool.org/exiftool_pod.html.
```
Comment (barasher) [inline exiftool.go:189]: The documentation does not fit godoc specifications. The comment should start with the name fonction.
https://blog.golang.org/godoc

**Comment by barasher** [comment]
Just a little godoc fix and that will be perdect :)

**Comment by Kjeldgaard** [comment]
Didn't know that was a thing :) It should be fixed now.

**Comment by barasher** [comment]
Thanks a lot @Kjeldgaard , I'll merge this feature tomorrow and release a new version

**Comment by Kjeldgaard** [comment]
You're welcome. Glad I could contribute to go-exiftool.

**Pull request merged**